librepo/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ TARGET_LINK_LIBRARIES(librepo
                         ${LIBCRYPTO_LIBRARIES}
                         ${GPGME_VANILLA_LIBRARIES}
                         ${GLIB2_LIBRARIES}
+                        ${GIO2_LIBRARIES}
```

Comment:
`GIO2_LIBRARIES` variable is uninitialized, so the project is not linked with gio library. Thus the `/usr/bin/ld: ../librepo/librepo.so.0: undefined reference to g_network_monitor_get_default` linker error (see the copr build log in Check tab - https://copr.fedorainfracloud.org/coprs/rpmsoftwaremanagement/CI-librepo-pr264/build/4800193)

One solution - drop this line and change https://github.com/rpm-software-management/librepo/blob/66d4f81d6bc75215179ff2517be4338ee50cccb6/CMakeLists.txt#L31 line to
`PKG_CHECK_MODULES(GLIB2 glib-2.0>=2.28 gio-2.0 REQUIRED)`